WordPress/src/main/java/org/wordpress/android/models/ReaderPost.java

```diff
@@ -205,6 +210,10 @@ public static ReaderPost fromJson(JSONObject json) {
         return post;
     }
 
+    public boolean hasImages() {
+        return hasText() && text.contains("<img ");
+    }
+
```

Comment:
I see you moved the `post.hasText() && post.text.contains("<img")` conditions to the new `hasImages()` method.  The second condition is changed from `"<img"` to `"<img "` in the new method.  I think using `"<img "` is a better check, but I just wanted to make sure that was intentional.